NExtends/Primitives/String.extensions.cs

```diff
@@ -9,555 +9,577 @@
+        public static String XLSProtect(this String value)
+        {
+            if (value != null)
+            {
+                String text = value.Replace("<br>", "\n")
+                                    .Replace("\r\n", " ")
+                                    .Replace("\n", " ").Replace("\"", "\"\"");
+
+                return @"""" + text + @"""";
+            }
+            else
+            {
+                return "";
+            }
+        }
+
+        /// <summary>
+        /// Pour faire un lower case sur un tableau de string
+        /// </summary>
+        /// <param name="array"></param>
+        /// <returns></returns>
+        public static string[] ToLower(this string[] array)
+        {
+            return array.Select(s => s.ToLower()).ToArray();
+        }
+
+        /// <summary>
+        /// String replacement with case insensivity support
+        /// </summary>
+        /// <param name="originalString"></param>
+        /// <param name="oldValue">The string to be replaced.</param>
+        /// <param name="newValue">The string to replace all occurrences of oldValue.</param>
+        /// <param name="comparisonType"></param>
+        /// <returns></returns>
+        public static string Replace(this string originalString, string oldValue, string newValue, StringComparison comparisonType)
+        {
+            // source
+            // http://stackoverflow.com/questions/5549426/is-there-a-case-insensitive-string-replace-in-net-without-using-regex
+
+            var startIndex = 0;
+            while (true)
+            {
+                startIndex = originalString.IndexOf(oldValue, startIndex, comparisonType);
+                if (startIndex == -1)
+                {
+                    break;
+                }
+
+                originalString = originalString.Substring(0, startIndex) + newValue + originalString.Substring(startIndex + oldValue.Length);
+
+                startIndex += newValue.Length;
+            }
+
+            return originalString;
+        }
+
+        public static void AppendLineFormat(this StringBuilder sb, string format, params object[] args)
+        {
+            sb.AppendLine(string.Format(format, args));
+        }
+
+        /// <summary>
+        /// http://stackoverflow.com/questions/472906/net-string-to-byte-array-c-sharp
+        /// </summary>
+        /// <param name="str"></param>
+        /// <returns></returns>
+        public static byte[] ToByteArray(this string str)
+        {
+            byte[] bytes = new byte[str.Length * sizeof(char)];
+            System.Buffer.BlockCopy(str.ToCharArray(), 0, bytes, 0, bytes.Length);
+            return bytes;
+        }
+
+        /// <summary>
+        /// Annee10 => 10
+        /// Annee => 1
+        /// </summary>
+        /// <param name="str"></param>
+        /// <param name="defaultValue (optional)"></param>
+        /// <returns></returns>
+        public static int GetOperand(this string str, int defaultValue = 1)
```

Comment:
don't understand this one. Still used ?

Reply:
Indeed, all the "Operand" methods are non generic ones, it's seems specific to a special functional case. I think we should remove them